generators/cypress/templates/src/test/javascript/cypress/e2e/account/logout.cy.ts.ejs

```diff
@@ -26,7 +26,7 @@ describe('logout', () => {
   const username = Cypress.env('E2E_USERNAME') ?? 'user';
   const password = Cypress.env('E2E_PASSWORD') ?? 'user';
 
-  it<%- clientFrameworkReact ? '.skip' : '' %>('go to home page when successfully logs out', () => {
+  it('go to home page when successfully logs out', () => {
     cy.login(username, password);
     cy.visit('');
     cy.clickOnLogoutItem();
```

Comment:
The wait might need to be increased since I think React does a redirect and takes a bit longer.